<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Member</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelKdMember">
      <property name="text">
       <string>Kode Member</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="editKdMember"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelNmMember">
      <property name="text">
       <string>Nama Member</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="editNmMember"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelEmail">
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="editEmail"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelHpTelp">
      <property name="text">
       <string>No. HP / Telepon</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="editHpTelp"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelKdKota">
      <property name="text">
       <string>Kode Kota </string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="editKdKota"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnSimpan">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>SIMPAN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHapus">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>140</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>HAPUS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUbah">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>UBAH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBersih">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>170</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>BERSIH</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editCari">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>401</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Ketik untuk mencari...</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabelMember">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>401</width>
     <height>171</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID Member</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nama</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Email</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>No. Telp</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Kota</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
